pkg/query-service/app/querier/v2/helper.go

```diff
@@ -0,0 +1,306 @@
+package v2
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	logsV3 "go.signoz.io/signoz/pkg/query-service/app/logs/v3"
+	metricsV3 "go.signoz.io/signoz/pkg/query-service/app/metrics/v3"
+	metricsV4 "go.signoz.io/signoz/pkg/query-service/app/metrics/v4"
+	tracesV3 "go.signoz.io/signoz/pkg/query-service/app/traces/v3"
+	"go.signoz.io/signoz/pkg/query-service/cache/status"
+	"go.signoz.io/signoz/pkg/query-service/constants"
+	v3 "go.signoz.io/signoz/pkg/query-service/model/v3"
```

Comment:
Is it recommended to use create a new model per version?

Reply:
Yes, that would be an ideal approach. We are using the same model for traces and logs as well. If we were to add a new version it would just apply for metrics but no change in the traces & logs. I was aiming for co-existence that doesn't confuse or require any changes from the traces/logs side.

Reply:
ok